nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/api/AccessResource.java

```diff
@@ -657,11 +658,12 @@ public Response createAccessTokenFromTicket(@Context HttpServletRequest httpServ
 
                 final String expirationFromProperties = properties.getKerberosAuthenticationExpiration();
                 long expiration = FormatUtils.getTimeDuration(expirationFromProperties, TimeUnit.MILLISECONDS);
-                final String identity = authentication.getName();
-                expiration = validateTokenExpiration(expiration, identity);
+                final String rawIdentity = authentication.getName();
+                expiration = validateTokenExpiration(expiration, rawIdentity);
```

Comment:
I think the logging in `validateTokenExpiration` should use `mappedIdentity` for better traceability.